barretenberg/cpp/src/barretenberg/aztec_ivc/aztec_ivc.cpp

```diff
@@ -45,6 +48,9 @@ void AztecIVC::accumulate(ClientCircuit& circuit, const std::shared_ptr<Verifica
         instance_vk = std::make_shared<VerificationKey>(prover_instance->proving_key);
     }
 
+    // Store whether the present circuit is a kernel
+    instance_vk->databus_propagation_data.is_kernel = (circuit_count % 2 == 0);
```

Comment:
nit: comment on why every 2nd circuit is a kernel could be nice

Reply:
thanks, left more explanation